app/api/export/exportConsentApi.ts

```diff
@@ -5,7 +5,7 @@ const exportConsentApi = async (
   consent: boolean,
   code: number,
 ): Promise<void> => {
-  const endpoint = `${authority.ingest_url}/consent`;
+  const endpoint = `${authority.public_api}/consent`;
```

Comment:
lt might make sense to group files by api rather than by endpoint, looks like theres some multiple places we need to updated the same thing.  This would also allow for pulling out helpers like defaultHeaders and baseUrl.